### ISSUE TITLE
More detalization.

### ISSUE THREAD

**Issue by gummybear** [zaidalyafeai/zaidalyafeai.github.io#6]
Can you help me, which parameter in the code changes the detail of the resulting image. Suppose that by reducing the productivity.

In pix2pix project.

**Comment by zaidalyafeai** [comment]
It is not trivial to increase the resolutions of the generated images. You would have to train on high res images but the model might be big and you wont be able to run in the browser.

**Comment by gummybear** [comment]
It's a pity. )

**Comment by gummybear** [comment]
А month later... )

Do you have a big model for pix2pix/scene_model? More than 50 MB. I do not necessarily have high speed work. Hour of work per frame - normal. )

If not, please tell me how to generate this model and on which data set. Thanks in advance.

**Comment by zaidalyafeai** [comment]
No, I don't have a larger model but you can train your own model. Follow the instructions [here](https://github.com/zaidalyafeai/zaidalyafeai.github.io/tree/master/pix2pix). To increase the size of the model use ngf = ndf = 64 for instance.
